api.go

```diff
@@ -270,8 +270,9 @@ type (
 	UploadTaskParams struct {
 		URL        string `json:"url,omitempty"`
 		Encryption struct {
-			Key       string `json:"key"`
-			Algorithm string `json:"algorithm,omitempty"`
+			EncryptedKey string `json:"encryptedKey,omitempty"`
```

Comment:
Should we have the public key used for encryption as well? Would be future-proof and safer